src/FacadeAccountCreation/FacadeAccountCreation.Core/Services/CompaniesHouse/CompaniesHouseLookupService.cs

```diff
@@ -17,7 +17,10 @@ public CompaniesHouseLookupService(HttpClient httpClient)
 
     public async Task<CompaniesHouseResponse?> GetCompaniesHouseResponseAsync(string id)
     {
-        var response = await _httpClient.GetAsync($"{CompaniesHouseEndpoint}/{id}");
+        var uriBuilder = new UriBuilder($"{CompaniesHouseEndpoint}/{Uri.EscapeDataString(id)}");
+        string endPoint = "CompaniesHouse" + uriBuilder.Path;
```

Comment:
Why are you putting CompaniesHouse in front? This looks like a different endpoint

Reply:
UriBuilder is changing URI Host to lowercase companieshouse
and it is expecting CompaniesHouse/companies/id